lib/metrics/names.js

```diff
@@ -169,7 +169,8 @@ const AI = {
   STREAMING_DISABLED: `${SUPPORTABILITY.NODEJS}/ML/Streaming/Disabled`,
   EMBEDDING: 'Llm/embedding',
   COMPLETION: 'Llm/completion',
-  TOOL: 'Llm/tool'
+  TOOL: 'Llm/tool',
+  AGENT: 'Llm/agent'
```

Comment:
Heads up: I didn't understand that LangChain has some sort of "agent" entity when I prepared the CDD. I'm not sure if `Llm/agent` is correct here. But I also don't know what else would be appropriate.

Looking at the spec, it might be that these should be: `Llm/completion/LangChain/invoke`. But I'm not sure.

Maybe @bizob2828 has more insight.